Source/Engine/ParselessPhraseDB.h

```diff
@@ -50,6 +50,11 @@ class ParselessPhraseDB {
 
     const char* findFirstMatchingLine(const std::string_view& key);
 
+    // Find the rows that (prefix-)match the value, useful for returning all the
+    // ways a phrase or character can be pronounced. Note that this is a potentially-
```

Comment:
No action required. I think this is fine. Linear-scanning 150k lines of data for a few times per system service invocation is acceptable for the use case of this PR.